testData/solutions/win32.json5

```diff
@@ -6506,6 +6506,23 @@
             }
         },
         "configure": [
+            // Remove current system wallpaper files.
+            {
+                "type": "gpii.windows.rm",
```

Comment:
I assume we will need mocks for this action in universal otherwise this solution will no longer pass through integration tests.

Reply:
Weirdly I was expecting this to fail, and the Windows to pass, the Windows one have failed with a spurious fail due to process killing timeout probably, and this have passed.